hibernate5/src/test/java/com/baeldung/hibernate/MultiTenantHibernateIntegrationTest.java

```diff
@@ -7,6 +7,8 @@
 
```

Comment:
Couple of things here:
1) it does not compile -- looks like the junit dependency was removed from the pom at some point, as was the build configuration.
2) in a test, you typically do not catch Exceptions; instead, declare the thrown exceptions on the test method